contribute/en/po4a-Setup-Explained.md

```diff
@@ -0,0 +1,117 @@
+
+4- The `po4a-create-all-targets.sh` script is run to create the translated .md files. They are not pushed, but they need to be created to be deployed to the website. The 'Statistics.md' file is also created.
+
+5- The website is built and deployed.
+
+6- A separate job, 'sync-branch', waits for the previous job to finish and then syncs 'release' in its current state to 'next-release'.
+
+**add-lang.yml**:
+This script is for adding a new language. It is triggered when an issue is opened containing a language code in either [xx] or [xx_XX] format in the title (e.g. "Add Portuguese [pt]" or "Add Brazilian Portuguese [pt_BR]") _and then_ the "new language" label is added. Summary:
+
+1- Checks whether the label added is "new language". If it is not, the whole workflow is skipped.
+
+2- Checks the format of the language code in the title and only accepts the format mentioned above. If it doesn't match, an error message is displayed and the script exits.
+
+3- Checks for the po4a cache as above.
+
+4- Creates a new directory in `_translator-files/po/` with the language code taken from the title.
+
+5- Runs `po4a-update-templates.sh` to populate the new folder with its .po files.
+
+6- Creates the target .md files with `po4a-create-all-targets.sh` and the 'Statistics' file.
+
+7- Builds the site but doesn't deploy nor zip/upload it - this and the previous step are just a check to verify that it builds properly.
+
+**jekyll.yml**:
+This is the more complex script. A distinction between 'pull request' and 'push' events will be made to better understand the flow of steps for each case:
+
+- **Pull Request events**. Applicable to both 'release' and 'next-release'; the steps are the same for both assuming the same PR content:
+
+    1- dorny/paths-filter@v2 checks if English .md files have changed/been added (but doesn't determine whether the script exits or not - see below).
+
+    2- Checks po4a cache (as above).
+
+    3- If the above filter is 'true', update/add .po files with `po4a-update-templates.sh`. If not, skip.
+
+    4- The output from the previous step is never pushed (even if it is 'true') when the trigger is a PR, as this would mean pushing the changes to the contributor's repository, which in turn would require them to enable GitHub actions beforehand. As this cannot be assumed to always be the case, it is left until their PR is merged and then the output is pushed to the Jamulus repository.
+
+    5- Translated .md files are created with `po4a-create-all-targets.sh`.
+
+    6- Site is built, zipped and uploaded as an artifact. This is so the contributor can download and view the site locally, to check translations as they would appear on the website or the overall appearance of the website if other content has been edited (config files, images, etc.) before the PR is merged. Only happens if the trigger was a PR.
```

Comment:
Maybe give a hint that the user should extract the .zip file and run it either with a local web server or jekyll serve. Just double-clicking on index.html doesn’t work.